comps/asr/src/integrations/dependency/whisper/whisper_server.py

```diff
@@ -106,7 +106,7 @@ async def audio_transcriptions(
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--host", type=str, default="0.0.0.0")
-    parser.add_argument("--port", type=int, default=7066)
+    parser.add_argument("--port", type=int, default=os.getenv("WHISPER_PORT", 7066))
```

Comment:
The internal container server port can be forwarded to any arbitrary host port. Why we need this also to be configurable?

Reply:
I see your point. Our intention with `WHISPER_PORT` and `ASR_PORT` was to enable full customization to the user, but as you pointed out, hard-coding the internal server ports doesn't really matter because they can be forwarded to different external ports.

The change to `ASR_PORT` is now irrelevant because it's been removed from GenAIExamples, so we will revert that change. Let me know if you prefer to have us revert `WHISPER_PORT` configurability and we'll do the same for it.

Reply:
Hi @mhbuehler , I think this should also be reverted, to keep redundant env variables as few as possible. Also I think even if users want to run the python script directly, they should not use an env variable like WHISPER_PORT but CAN use `python whisper_server.py --port XXX` for port customization.

Reply:
@Spycsh  Thanks, that makes sense. I pushed an update to revert this change.